content/en/docs/concepts/signals/logs.md

```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
-design, specifies a new API, and provides full implementations of this API in
-multiple language SDKs.
+## OpenTelemetry Logs
 
-OpenTelemetry's approach with logs is different. Because existing logging
-solutions are widespread in language and operational ecosystems, OpenTelemetry
-acts as a "bridge" between those logs, the tracing and metrics signals, and
-other OpenTelemetry components. In fact, the API for logs is called the "Logs
-Bridge API" for this reason.
+OpenTelemetry does not define a bespoke API or SDK to create logs. Instead,
+OpenTelemetry logs are simply the existing logs you already have from a logging
+framework or infrastructure component. Under the covers, OpenTelemetry SDKs and
+autoinstrumentation utilize several components to automatically correlate logs
+with [traces](/docs/signals/traces).
 
-The [logs specification][] contains more details on this philosophy.
+OpenTelemetry's support for logs is designed to be maximally compatible with
+what you already have, providing capabilities to wrap those logs with additional
+context and a common toolkit to parse and manipulate logs into a common format
+across many different sources.
 
-To understand how logging in OpenTelemetry works, let's look at a list of
-components that will play a part in instrumenting our code.
+### OpenTelemetry Logs in the OpenTelemetry Collector
 
-## Log Appender / Bridge
+The [OpenTelemetry Collector](/docs/collector) provides several tools to work
+with logs:
+
+- Several receivers which parse logs from specific, known sources of log data
+- The filelogreceiver, which reads logs from any file and provides features to
+  parse them from different formats or use a regular expression
+- Processors like the transformprocessor which lets you parse nested data,
+  flatten nested structures, add/remove/update values, and more
+- Exporters that let you emit log data in a non-OpenTelemetry format
+
+Often times, the first step in adopting OpenTelemetry involves deploying a
+Collector as a general-purposes logging agent.
+
+### OpenTelemetry Logs for Applications
+
+In applications, OpenTelemetry logs are created with any logging library or
+built-in logging capabilities. When you add autoinstrumentation or activate an
+SDK, OpenTelemetry will automatically correlate your existing logs with any
+active trace and span, wrapping the log body with their IDs. In other words,
+OpenTelemetry will automatically correlate your logs!
+
+### Language Support
+
+Logs are a [stable](/docs/specs/otel/versioning-and-stability/#stable) signal in
+the OpenTelemetry specification. For the individual language specific
+implementations of the Logs API & SDK, the status is as follows:
+
+{{% signal-support-table "logs" %}}
+
+## Structured, Unstructured, and Semistructured Logs
+
+OpenTelemetry does not distinguish between structured and unstructured logs.
+However, it is vitally important to understand the different kinds of formats
+that log data can manifest in.
```

Comment:
```suggestion
OpenTelemetry does not distinguish between structured and unstructured logs.
However, it is vitally important to understand the different kinds of formats
that log data can manifest in.
```

If OTel doesn't distinguish log types, why understanding the different kinds matter? I think this intro would benefit from a rewrite that focuses on why one should continue reading.